modules/cluster-logging-deploy-label.adoc

```diff
@@ -5,13 +5,9 @@
 [id="cluster-logging-deploy-label_{context}"]
 = Labeling nodes
 
-At 100 nodes or more, it is recommended to first pre-pull the logging images
-from the registry.
-After deploying the logging pods (Elasticsearch, Kibana, and
-Curator), node labeling should be done in steps of 20 nodes at a time. For
-example:
+At 100 nodes or more, it is recommended to first pre-pull the logging images from the registry. After deploying the logging pods (Elasticsearch and Kibana), node labeling should be done in steps of 20 nodes at a time. For example:
```

Comment:
Not sure if the parenthesis is frowned upon for these documents. But if so, consider using commas instead:
`After deploying the logging pods, Elasticsearch and Kibana, node labeling should be done in steps of 20 nodes at a time.`